desidlas/training/training.py

```diff
@@ -0,0 +1,397 @@
+""" Methods for training the Model"""
+#!python
+
+"""
+0. Update all methods to TF 2.1 including using tf.Dataset
+"""
+import numpy as np
+import math
+import re, os, traceback, sys, json
+import argparse
+import tensorflow as tf
+
+from tensorflow.python.framework import ops
+ops.reset_default_graph()
+
+from model import build_model
+
+from tensorflow.compat.v1 import ConfigProto
+from tensorflow.compat.v1 import InteractiveSession
+
+config = ConfigProto()
```

Comment:
What are these lines doing?  Add a comment.

Reply:
These lines are used for parameter configuration for tensorflow session, I have added inline comments.